downloads/pt/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: cheat-sheet
+title: Folheto de ajuda para Git do GitHub (pt-BR)
+byline: Git é um sistema de controlo de versão distribuído open source que facilita ações com o GitHub no seu notebook ou desktop. Este folheto de ajuda resume instruções frequentemente usadas na linha de comando do Git para referência rápida.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Instalar o Git
+
+O GitHub fornece clientes desktop que incluem um interface gráfico para as ações mais comuns em repositórios. Estes clientes também incluem ferramentas da linha de comandos para cenários mais avançados. Em ambos os casos, os clientes são actualizados automáticamente.
```

Comment:
I'll try to give it a Portuguese spin. Some of the original as I read it I could see it going both ways. But it's good that you say that, I'll just re-write as it will come more natural.